tests/test_numpy_array.py

```diff
@@ -435,6 +435,45 @@ def test_index_using_ellipsis():
     assert a.shape == (6,)
 
 
+@pytest.mark.parametrize("forcecast", [False, True])
+@pytest.mark.parametrize("contiguity", [None, 'C', 'F'])
```

Comment:
In terms of integers, this would translate to `0`, `1` and `2`. Where's `c_style | f_style`? That combination still makes no sense to me, but you've shown me that it is possible in gitter.

Reply:
Well, it's basically only an array with all dimensions 1. So I could add a `np.zeros(1)`, if you want? I just thought it would be even more complicated than it already is.

Should I?

Reply:
> Well, it's basically only an array with all dimensions 1.

So that's how it works - `[[[[0]]]]`.

> So I could add a np.zeros(1), if you want? I just thought it would be even more complicated than it already is.

Why not? It's not like we need to worry about the size of the tests.

Reply:
> So that's how it works - `[[[[0]]]]`.

You can't deny that that's technically contiguous in every way, right? ;-)

> Why not? It's not like we need to worry about the size of the tests.

As in: this test already exploded anyway?

Reply:
Done!